src/components/MenuBtn/MenuBtn.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './MenuBtn.module.scss';
+
+const MenuBtn = (props) => {
+  return (
+    <button className={styles.menu__btn} onClick={props.handleOnClick}>
+      {props.children}
+    </button>
+  );
+};
+
+export default MenuBtn;
```

Comment:
So I created component for menu buttons. Unfortunately at the moment I don't know how to implement them into Settings menu.

Reply:
In `Gameover.js`<MenuBtn handleOnClick={handleClick}>Start again</MenuBtn>` instead of `<button className={styles.button} onClick={handleClick}>
        Start again
      </button>`. Import is needed. You would need to add className and some logic behind MenuBtn styles.